### PR TITLE
[3.1.0] Add required steps when enabling user signup workflow for multi tenants

### LINKED ISSUE #2130: [APIM 3.2] Missing steps when enabling usersignup workflow for tenants
**Description:**
Following guide[1] should be updated with the extra steps required if there are multiple tenants and you need to enable this workflow for each tenant. 

1. At tenant admin's login in carbon console  https://apim.docs.wso2.com/en/latest/develop/customizations/customizing-the-developer-portal/enabling-or-disabling-self-signup/#enabling-self-signup should be configured.
2. At tenant admin's login in carbon console following file (/_system/governance/apimgt/applicationdata/workflow-extensions.xml) should be updated with the new workflow executor. 

Basically need a note stating that in a multi tenant environment, following steps should be followed.
**Suggested Labels:**

**Suggested Assignees:**

**Affected Product Version:**

**OS, DB, other environment details and versions:**    

**Steps to reproduce:**

**Related Issues:**

### PR DESCRIPTION
**Pull request by Jayanie**
## Purpose

- This PR fixes #2130
- Add a note providing steps that need to be performed when there are multiple tenants.
- Corrects "Register now" image

### CODE REVIEW

**Review by pubudu538 — APPROVED** [review]

**Review by pubudu538 — APPROVED** [review]

File: en/docs/develop/customizations/adding-a-user-signup-workflow.md [hunk 1]
```diff
@@ -182,6 +182,10 @@ This section explains how to attach a custom workflow to the user signup operati
 
 #### Engaging the WS Workflow Executor in the API Manager
 
+!!! important
+    When there are multiple tenants, first enable self sign up for tenant admin for each tenant by following the steps mentioned in [Enabling self signup]({{base_path}}/develop/customizations/customizing-the-developer-portal/enabling-or-disabling-self-signup/#enabling-self-signup). 
+    Then to add a user sign-up workflow with regard to the Developer Portal, follow the below instructions from step 1 after sign in to the API-M Management Console as tenant admin for each tenant.
```
Comment (Ketharan) [inline en/docs/develop/customizations/adding-a-user-signup-workflow.md:187]: Better skip short forms like 'API-M' in docs, check other places as well.

File: en/docs/develop/customizations/adding-a-user-signup-workflow.md [hunk 2]
```diff
@@ -204,13 +208,9 @@ This section explains how to attach a custom workflow to the user signup operati
...
 3.  Go to the Developer Portal Web interface of API Manager and sign up / register as a new user.
-<html>
-    <body>
-        <div>
-            <img src="../../../../assets/img/learn/register-now.png" alt="Register now option" width="600"/>
-        </div>
-    </body>
-</html>
+
+    ![Register now option](../../assets/img/learn/register-now.png)
```
Comment (Ketharan) [inline en/docs/develop/customizations/adding-a-user-signup-workflow.md:212]: shall we refer to the complete path including the base path?

Comment (Jayanie) [reply en/docs/develop/customizations/adding-a-user-signup-workflow.md:212]: Fixed with 10584cd96c089244ce8914132e6b047ff59d5453

File: en/docs/develop/customizations/adding-a-user-signup-workflow.md (hunk 1 above)
Comment (Jayanie) [reply en/docs/develop/customizations/adding-a-user-signup-workflow.md:187]: Fixed with 10584cd96c089244ce8914132e6b047ff59d5453

**Pull request merged**